docs/vsphere-golden-image.md

```diff
@@ -158,13 +165,21 @@ where `<path/to/unattend.xml>` is the path to the customized answer file.
 Note: There is [a limit](https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/sysprep--generalize--a-windows-installation#limits-on-how-many-times-you-can-run-sysprep)
 on how many times you can run the `sysprep` command.
 
-## 7. Set up the virtual machine golden image
+## 7. Set up the virtual machine template
 
 Once the `sysprep` command completes the Windows virtual machine will power off. 
 
-You **must not** use this Windows virtual machine anymore. 
+You **must not** use or power on this virtual machine anymore.
+
+Next, you need to convert this virtual machine, in Power-Off status, to a Template as shown in Figure 1.
+
+|![Convert VM to Template](images/vcenter-vm-to-template.png)|
+|:---|
+|Figure 1. Steps to convert a VM to Template in vCenter|
 
-Add the cloned Windows virtual machine as a template in the [machineset](../README.md#configuring-windows-instances-provisioned-through-machinesets).
+## 8. Using the virtual machine template
+
+In order to use the recently created template, enter the template name in the [machineset](../../README.md#configuring-windows-instances-provisioned-through-machinesets).
```

Comment:
The machineset link doesn't work. Perhaps it should point back to `../README.md#configuring-windows-instances-provisioned-through-machinesets`?

Reply:
Fixed in https://github.com/openshift/windows-machine-config-operator/pull/515/commits/2a33a4a1762187c6005b5ee9176a5fb371d8b9c3